src/style.css

```diff
@@ -0,0 +1,91 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  --main-color: #3d4e61;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.search-bar__form {
+  margin: 0 8px;
```

Comment:
actually, you can remove this margin and remove margin reset from [body](https://github.com/mate-academy/layout_search-bar-airbnb/pull/3442/files#diff-1104bdfdc4d3b01a85d7af489598121125db6bc016db2a48a083ef654c97b0f5R2), and it will have the same result, I think

Reply:
because default body margins is 8px